templates/archive.html

```diff
@@ -1,21 +1,21 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"8rem","bottom":"8rem"}}},"layout":{"inherit":true}} -->
-<main class="wp-block-group" style="margin-top:8rem;margin-bottom:8rem">
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var(--preset--spacing--70)","bottom":"var(--preset--spacing--70)"}}},"layout":{"inherit":true}} -->
+<main class="wp-block-group" style="margin-top:var(--preset--spacing--70);margin-bottom:var(--preset--spacing--70)">
 	<!-- wp:query-title {"type":"archive","align":"wide","style":{"spacing":{"margin":{"bottom":"6.4rem"}}}} /-->
 	<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"displayLayout":{"type":"flex","columns":3},"align":"wide","layout":{"inherit":false}} -->
 	<div class="wp-block-query alignwide">
 		<!-- wp:post-template {"align":"wide"} -->
 			<!-- wp:post-featured-image {"isLink":true,"align":"wide"} /-->
-			<!-- wp:post-title {"isLink":true,"style":{"spacing":{"margin":{"top":"var(--preset--spacing--60)","bottom":"var(--preset--spacing--60)"}}}} /-->
+			<!-- wp:post-title {"isLink":true,"style":{"spacing":{"margin":{"top":"var(--preset--spacing--40)","bottom":"var(--preset--spacing--40)"}}}} /-->
```

Comment:
I think these are missing the `--wp` prefix, also it made me think — should we apply this same spacing to the Latest Posts section on the Home template? 

```suggestion
			<!-- wp:post-title {"isLink":true,"style":{"spacing":{"margin":{"top":"var(--wp--preset--spacing--40)","bottom":"var(--wp--preset--spacing--40)"}}}} /-->
```

Reply:
Ugh, yes, good spot!

> should we apply this same spacing to the Latest Posts section on the Home template?

Yeah, I think that would work 👍